roles/code_server/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+#########################################################################
+# Title:            Sandbox: code-server                                #
+# Author(s):        astroMD, kowalski                                   #
+# URL:              https://github.com/saltyorg/Sandbox                 #
+# --                                                                    #
+#########################################################################
+#                   GNU General Public License v3.0                     #
+#########################################################################
+---
+- name: Import Legacy task
+  ansible.builtin.import_tasks: "subtasks/legacy.yml"
```

Comment:
Move the conditional check to here and use `when: code_server_migrate_coder is defined` so we can use the var without having to define it beforehand